tests/phpunit/tests/script-modules/wpScriptModules.php

```diff
@@ -11,7 +11,8 @@
  *
  * @coversDefaultClass WP_Script_Modules
  */
-class Tests_WP_Script_Modules extends WP_UnitTestCase {
+class Tests_WpScriptModules extends WP_UnitTestCase {
```

Comment:
I think the name of the subfolder should also be included here:
```suggestion
class Tests_Script_Modules_WpScriptModules extends WP_UnitTestCase {
```

Reply:
Fixed